js/core/src/reflection.ts

```diff
@@ -169,48 +164,30 @@ export class ReflectionServer {
           return;
         }
         if (stream === 'true') {
-          const result = await newTrace(
-            { name: 'dev-run-action-wrapper' },
```

Comment:
Why did we have this to begin with? Just trying to consider if it's consequence free to remove.

Reply:
we had it because we didn't have a clean way to get trace id from an action (what this PR adds). We had to wrap it in a separate span from which we extracted the trace id, then we marked that span as 'genkit-dev-internal' and dev ui has code to ignore those spans when it sees them.